doc/user_guide/encryption.rst

```diff
@@ -23,7 +25,8 @@ Certification verification
 - Exasol Docker uses self-signed SSL certificate by default. You may generate a proper SSL certificate and use it via editing of EXAConf file. More details are available on the `GitHub page <https://github.com/exasol/docker-db>`_.
 - Exasol SAAS running in the cloud uses proper certificate generated by public certificate authority. It does not require any extra setup.
 
-Certificate verification is disabled by default for connections with username and password. Certificate verification is enabled by default for connections with username and OpenID token.
+(security risk) To disable strict certification verification, set ``websocket_sslopt`` to ``{"cert_reqs": ssl.CERT_NONE}``.
+This might be needed for specific use cases (e.g. Docker container, "on-premises" Exasol setup).
```

Comment:
Probably you mean that disabling verification should only done for development or testing, but not in production environments.

Reply:
Yes, that could have been better worded. Lmk what you think of the updated text.